src/styles/themes/default.js

```diff
@@ -83,6 +82,8 @@ const darkTheme = {
     QRLogo: colors.green400,
     starDefaultBG: 'rgb(254, 228, 94)',
     loungeAccessOverlay: colors.blue800,
+    textBrand: colors.green400,
```

Comment:
Not sure how I feel about this name... maybe it's better to just have a series of helper classes like `colorGreen400` and then just call that? Otherwise textBrand doesn't feel very descriptive and I'm not sure how/where else we'd use it?